cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -95,13 +136,24 @@ template <typename InputIteratorT,
           typename ScanOpT,
           typename InitValueT,
           typename OffsetT,
-          typename AccumT     = ::cuda::std::__accumulator_t<ScanOpT,
-                                                             cub::detail::value_t<InputIteratorT>,
-                                                             ::cuda::std::_If<std::is_same<InitValueT, NullType>::value,
-                                                                              cub::detail::value_t<InputIteratorT>,
-                                                                              typename InitValueT::value_type>>,
-          typename PolicyHub  = detail::scan::policy_hub<AccumT, ScanOpT>,
-          bool ForceInclusive = false>
+          typename AccumT       = ::cuda::std::__accumulator_t<ScanOpT,
+                                                               cub::detail::value_t<InputIteratorT>,
+                                                               ::cuda::std::_If<std::is_same<InitValueT, NullType>::value,
```

Comment:
```suggestion
                                                               ::cuda::std::_If<::cuda::std::is_same_v<InitValueT, NullType>,
```

Reply:
xref https://github.com/NVIDIA/cccl/pull/3398#discussion_r1918880186

Reply:
Please apply the change suggested above, since it saves template instantiations if we use only traits from `::cuda::std::*`

Reply:
Ah, thanks - good to know. I applied it as part of https://github.com/NVIDIA/cccl/pull/3398/commits/a6ed893bd0b370f718115cc34078ee06d6c1cc7b.